### PR TITLE
Disable console install when not 3.9 or newer

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://github.com/openshift/origin/issues/18573

/assign @sdodson 
@smarterclayton @stevekuznetsov

### CODE REVIEW

File: playbooks/init/basic_facts.yml [hunk 1]
```diff
@@ -74,4 +74,4 @@
   tasks:
   - set_fact:
       openshift_web_console_install: False
-    when: openshift_deployment_subtype == 'registry' or ( osm_disabled_features is defined and 'WebConsole' in osm_disabled_features )
+    when: not openshift.common.version_gte_3_9 or openshift_deployment_subtype == 'registry' or ( osm_disabled_features is defined and 'WebConsole' in osm_disabled_features )
```
Comment (sdodson) [inline playbooks/init/basic_facts.yml]: This fact is likely not available at this point in playbook execution, which is unfortunate as this is the best place to ensure all code paths skip console installation.

Try moving it https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/private/components.yml#L23

**Comment by spadgett** [comment]
Thanks @sdodson updated

**Comment by sdodson** [comment]
flake
/test install

**Pull request merged**